README.mdown

```diff
@@ -32,3 +32,20 @@ To run tests do:
     python2 bookshelf/api_v2/test_ec2.py
     python2 bookshelf/api_v2/test_rackspace.py
 
+There are also tests for the api_v3 that spin up GCE, Rackspace, and EC2
+instances. These tests require credentials to access GCE, Rackspace, and EC2.
+In an effort to attempt to reduce the amount of work to set up the
+configuration, they build on the existing acceptance.yml as described at:
+http://doc-dev.clusterhq.com/gettinginvolved/appendix.html#acceptance-testing-configuration.
+
+Note that instead of starting from that configuration, you can start with an
+empty yml file and the tests will print out what configuration you need to add
+in to your yaml file in order to run the tests.
+
```

Comment:
I got the idea from flocker's tests. Skipping tests with actionable messages about what you have to do to actually run the test I consider to be a really nice developer experience.